content/developer/tutorials/discover_js_framework/01_owl_components.rst

```diff
@@ -284,15 +291,18 @@ The final touch is to let the user delete a todo.
    :scale: 70%
    :align: center
 
-10. Generic components with slots
-=================================
+.. _tutorials/discover_js_framework/generic_card:
+
+10. Generic card with slots
+===========================
 
 Owl has a powerful `slot <{OWL_PATH}/doc/reference/slots.md>`_ system to allow you to write generic
 components. This is useful to factorize the common layout between different parts of the interface.
 
 .. exercise::
 
-   #. Write a `Card` component using the following Bootstrap HTML structure:
+   #. Insert a new `Card` component between the `Counter` and `Todolist` components. Use the
+      following Bootstrap HTML structure for the card:
 
       .. code-block:: html
 
```

Comment:
I just noticed that there is no indentation in the `card-body` div in the html below. Maybe we can fix it in this PR